<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurnoutDlg</class>
 <widget class="QDialog" name="TurnoutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>317</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Val 1: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edV1">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Val 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edV2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbB1Mode">
     <property name="title">
      <string>B1 mode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbB1Input">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbB1Output">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkDS54Aux">
     <property name="text">
      <string>DS54 Aux Input</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkClosedOn">
     <property name="text">
      <string>Closed ON</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkOutputOn">
     <property name="text">
      <string>Output ON</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbAddr"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkInputHigh">
     <property name="text">
      <string>Input High (&gt; +6V)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblHelp">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
